module/project/templates/helper.js

```diff
@@ -106,7 +106,7 @@ module.exports = class ProjectTemplatesHelper {
                 let recommendedFor = {};
 
                 if( roleIds.length > 0 ) {
-
+                    roleIds = UTILS.normalizeToLower(roleIds)
```

Comment:
@Prajwal17Tunerlabs Hope there is no other call happening directly to userRoles collection from project service. please check and confirm here once